ecs-cli/modules/config/config.go

```diff
@@ -26,7 +26,10 @@ import (
 
 // TODO: This needs a better home.
 
-const RegionFlag = "region"
+const (
+	RegionFlag                  = "region"
+	EC2RoleProviderExpiryWindow = 5 * time.Minute
```

Comment:
This can be private, and it might make sense to reference where this time comes from (looks like it was the SDK defaults package).